2-mandatory/3-stocks.js

```diff
@@ -48,9 +64,17 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
-}
-
+        const priceChanges = [];
+      
+        for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+          const stockPrices = closingPricesForAllStocks[i];
+          const priceChange = stockPrices[stockPrices.length - 1] - stockPrices[0];
+          priceChanges.push(Number(priceChange.toFixed(2)));
+        }
+      
+        return priceChanges;
+      }
```

Comment:
Comment: Nice solution

Here is one with less code:
function getPriceChanges(closingPricesForAllStocks) {
    // TODO
      let changedPrices = [];
  for (let arrayWithPrices of closingPricesForAllStocks) {
    let changedPrice = arrayWithPrices[arrayWithPrices.length - 1] - arrayWithPrices[0];
    changedPrices.push(Number(changedPrice.toFixed(2)));
  }
  return changedPrices;
}